day00/ex04/ex04.md

```diff
@@ -0,0 +1,59 @@
+# Exercise 04 - Dot product
+
+|                         |                    |
+| -----------------------:| ------------------ |
+|   Turnin directory :    |  ex04              |
+|   Files to turn in :    |  dot.py            |
+|   Forbidden function :  |  np.dot()          |
+|   Remarks :             |  n/a               |
+
+**Objective:**
+You must implement the following formula as a function:  
+  
+$$
+x \cdot y = xy^{T} = \sum_{i = 1}^{m} x_i y_i
```

Comment:
I have some doubts on the vector representation, a vector should be an array of Nx1 shaped and not 1xN shape. I might be wrong.
In other case, it becomes x^{T}y instead of xy^{T}